pmm/main.go

```diff
@@ -205,10 +205,25 @@ func (a *Admin) ServerInfo() {
 
 // StartStopMonitoring start/stop system service by its metric type and name.
 func (a *Admin) StartStopMonitoring(action, svcType string) error {
-	if svcType != "linux:metrics" && svcType != "mysql:metrics" && svcType != "mysql:queries" && svcType != "mongodb:metrics" && svcType != "proxysql:metrics" {
-		return errors.New(`bad service type.
+	svcTypes := []string{
+		"linux:metrics",
+		"mysql:metrics",
+		"mysql:queries",
+		"mongodb:metrics",
+		"mongodb:queries",
+		"proxysql:metrics",
+	}
+	valid := false
+	for _, v := range svcTypes {
+		if v == svcType {
+			valid = true
```

Comment:
Just a cosmetic change. You don't need the `valid` variable. You can just `return nil` if you find the element and avoid the `if` below